tests/tasks/create_test_interfaces_with_dhcp.yml

```diff
@@ -43,8 +43,15 @@
       nmcli d set testbr managed false
     fi
     ip link set testbr up
-    ip addr add 192.0.2.1/24 dev testbr
-    ip -6 addr add 2001:DB8::1/32 dev testbr
+    timer=0
+    while ! ip a s testbr | grep -q 'inet [1-9]'
+    do
+        let "timer+=1"
+        if [ $timer -eq 30 ]; then break; fi
+        ip addr add 192.0.2.1/24 dev testbr
+        ip -6 addr add 2001:DB8::1/32 dev testbr
```

Comment:
is it correct to add the addresses again and again?

Reply:
The following is the experiment that I did on downstream machine, notice that the order between calling `ip link set testbr2 up` and `ip addr add 192.0.2.1/24 dev testbr2` matters in this situation. and calling `ip link set testbr2 up` may take a few seconds in downstream machine.  Even if we can call `ip link set testbr2 up` first and then call `ip addr add 192.0.2.1/24 dev testbr2`, racing issue will still happen. 

 
```
 [root@ci-vm-10-0-138-87 ~]# ip link add name testbr2 type bridge forward_delay 0
[root@ci-vm-10-0-138-87 ~]# ip a s testbr2
12: testbr2: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 62:fa:f8:fe:21:15 brd ff:ff:ff:ff:ff:ff
[root@ci-vm-10-0-138-87 ~]# ip addr add 192.0.2.1/24 dev testbr2
[root@ci-vm-10-0-138-87 ~]# ip a s testbr2
12: testbr2: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 62:fa:f8:fe:21:15 brd ff:ff:ff:ff:ff:ff
    inet 192.0.2.1/24 scope global testbr2
       valid_lft forever preferred_lft forever
[root@ci-vm-10-0-138-87 ~]# ip link set testbr2 up
[root@ci-vm-10-0-138-87 ~]# ip a s testbr2
12: testbr2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN group default qlen 1000
    link/ether 62:fa:f8:fe:21:15 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::60fa:f8ff:fefe:2115/64 scope link tentative 
       valid_lft forever preferred_lft forever


[root@ci-vm-10-0-138-87 ~]# ip a s testbr2
14: testbr2: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 86:39:6b:55:bd:05 brd ff:ff:ff:ff:ff:ff
[root@ci-vm-10-0-138-87 ~]# ip link set testbr2 up  && ip addr add 192.0.2.1/24 dev testbr2
[root@ci-vm-10-0-138-87 ~]# ip a s testbr2
14: testbr2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN group default qlen 1000
    link/ether 86:39:6b:55:bd:05 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::8439:6bff:fe55:bd05/64 scope link tentative 
       valid_lft forever preferred_lft forever

```